packages/webapp/.eslintrc.cjs

```diff
@@ -26,6 +28,31 @@ const config = {
 				caughtErrorsIgnorePattern: "^_",
 			},
 		],
+		"import/first": "error",
+    "import/newline-after-import": "error",
+    "import/no-duplicates": "error",
+		"simple-import-sort/imports": [
+			"error",
+			{
+				"groups": [
+					// Packages. `react` related packages come first.
+					[
+						"^react",
+						"^next",
+					],
+					// Internal packages.
+					["^@?\\w"],
+					// Parent imports. Put `..` last.
+					["^\\.\\.(?!/?$)", "^\\.\\./?$"],
+					// Other relative imports. Put same-folder imports and `.` last.
+					["^\\./(?=.*/)(?!/?$)", "^\\.(?!/?$)", "^\\./?$"],
+					// Style imports.
+					["^.+\\.s?css$"],
+					// Side effect imports.
+					["^\\u0000"]
```

Comment:
One rule that would be great is to make sure the internal imports, which we prefix with `src/` are in their own separate section (after the "internal packages"  would be ideal — we should also change that comment to actually say "external packages" (stuff like wagmi etc))